apollo-client/src/main/java/com/ctrip/framework/apollo/spring/annotation/SpringValueProcessor.java

```diff
@@ -124,20 +116,33 @@ protected void processMethod(Object bean, String beanName, Method method) {
       return;
     }
 
-    Set<String> keys = placeholderHelper.extractPlaceholderKeys(value.value());
+    doRegister(bean, beanName, method, value);
+  }
 
+  private void doRegister(Object bean, String beanName, Member member, Value value) {
+    Set<String> keys = placeholderHelper.extractPlaceholderKeys(value.value());
     if (keys.isEmpty()) {
       return;
     }
 
     for (String key : keys) {
-      SpringValue springValue = new SpringValue(key, value.value(), bean, beanName, method, false);
+      SpringValue springValue;
+      if (member instanceof Field) {
+        Field field = (Field) member;
+        springValue = new SpringValue(key, value.value(), bean, beanName, field, false);
+      } else if (member instanceof Method) {
+        Method method = (Method) member;
+        springValue = new SpringValue(key, value.value(), bean, beanName, method, false);
+      } else {
+        logger.error("@Value annotation current is only support to be used on methods or property, "
+            + "but used on {}", member.getClass());
+        return;
+      }
       springValueRegistry.register(beanFactory, key, springValue);
-      logger.info("Monitoring {}", springValue);
+      logger.debug("Monitoring {}", springValue);
```

Comment:
Is it necessary to change the logging level? I think it's ok to leave this as it only prints once and it may affect the user who relies on this log.

Reply:
Yes, it’s better to change to info logging level here.